### ISSUE TITLE
Default localization

### ISSUE THREAD

**Issue by tchauviere** [PrestaShopCorp/pagseguro#2]
Hi @cegoncalves,

Has a general comment, and since your latests commits seems to change a lot the module itself, it would be nice to make default in English, and add Brazilian only when Shop is in Brazilian.

Thanks!

**Comment by cegoncalves** [comment]
Our payment method works only in Brazil, so we see no point in keeping two languages since our target audience is 100% Brazilian.

**Comment by tchauviere** [comment]
Hi @cegoncalves,

This is a PrestaShop norm: http://doc.prestashop.com/display/PS16/Module+translation

This is asked because we have to review your module and unfortunately (for me) I don't speak Brazilian Portuguese at all, not making it easy for me to understand what's each menu is about.

But this is not the only reason, here's an example (which could impact Pagseguro itself, by loosing potential clients):

Let's say I'm an English based e-merchant selling goods only in Brazil, I want to use your module, but I can't read / understand it at all. I would probably move to another solution which has English support.

In conclusion, English is really important for us, just like it is for you.

**Comment by cegoncalves** [comment]
Hi @tchauviere

I believe we have solved this issue with the commit f038988c44e725f96e99b2df439fcbe273304daa

**Comment by Quetzacoalt91** [comment]
Hi @cegoncalves,

Can you also please update the content of the file _pagseguro.php_ ? Thanks

**Comment by Quetzacoalt91** [comment]
As @tchauviere said, the content should be written by default in English, and then translated in Brazilian. Thanks
